### DISCUSSION TITLE
German translation

### DISCUSSION THREAD

**Discussion by j6845** [LorisYounger/VPet#395] [Translate]
Hey, i saw the russian translation issue and would like to do it for the language german.

By reading the issue https://github.com/LorisYounger/VPet/issues/271
i am not fully sure about what is needed.

Can you provide all texts, needed to be translated for steam, so the issue can be resolved without much enquire? :)

**Comment by LorisYounger** [comment]
Translate files in `mod\0000_core\lang`
you can use mod maker or Manually creating MODs
The Core or other  translation mod increative workshops should give you a reference.
Here you can also download translation tools (demo) specifically written for translation.
https://github.com/LorisYounger/LinePutScript.Localization.WPF

for german. https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c
you can create a folder called `ge` in  `lang`folder for german truanslate

**Comment by LorisYounger** [comment]
Also, in github.  the game itself will not add languages other than English, Simplified Chinese, and Traditional Chinese
Because I am unable to confirm the text content and there are currently many versions of translation documents in the workshop, I cannot choose to make one of them an official translator.
You can add [Human Translation] in title that Let everyone know the advantages of your translation
You can also post it in the discussion Let everyone know that there is a French translation available
I will also release a special announcement in the future to inform everyone of the existing community translation content and location
Thank you for support of VPET project

**Comment by j6845** [comment]
Thanks, i will take a look at your tool!

I have already completed the german language for the game itself.
I was referring to the steam website, where information about the game is in english for german users.
